lib/devise_security_extension/models/password_expirable.rb

```diff
@@ -22,22 +22,18 @@ def need_change_password?
 
       # set a fake datetime so a password change is needed and save the record
       def need_change_password!
-        if self.class.expire_password_after.is_a? Fixnum or self.class.expire_password_after.is_a? Float
-          need_change_password
-          self.save(:validate => false)
+        if self.class.expire_password_after.is_a?(Numeric)
```

Comment:
`is_a?`, `kind_of?`, `respond_to?` are antipatterns—better to use duck typing and polymorphism.